lib/Syntax/README.md

```diff
@@ -559,3 +559,48 @@ Here's a handy checklist when implementing a production in the grammar.
 - Update `lib/Syntax/Status.md` if applicable.
 
 [NewSwiftBug]: https://bugs.swift.org/secure/CreateIssue!default.jspa)
+
+## Try libSyntax in Xcode
+
+Here's how to build a Swift command line tool in Xcode using libSyntax:
+1. Download the latest open source toolchain from swift.org:
+  [Trunk Development (master)](https://swift.org/download/#snapshots).
+2. Run the downloaded package installer.
+3. Start Xcode and specify the just-installed toolchain to use in
+`Xcode->Toolchains->Swift Development Snapshot...`
+4. Create a new Swift command line tool project in Xcode.
+5. In the project's build setting, specify two variables:
+    - Runpath search paths: `$(TOOLCHAIN_DIR)/usr/lib/swift/macosx`
+    - Library search paths: `$(TOOLCHAIN_DIR)/usr/lib/swift/macosx`
```

Comment:
The latter should already automatically be set, since that's where the standard library module lives. (The former makes sense because of static vs. dynamic linking. I'm actually not sure if this is guaranteed to work in general because you're mixing a statically-linked stdlib with the dynamically-linked libSwiftSyntax.dylib, but oh well.)